src/index.ts

```diff
@@ -164,6 +165,13 @@ const parameters: JupyterFrontEndPlugin<void> = {
         await console.sessionContext.changeKernel({ name: kernel });
       }
 
+      if (install) {
+        await console.sessionContext.ready;
+        console.inject('import micropip');
+        install[0].split(',').forEach(pckg => console.inject(`await micropip.install('${pckg}')`));
```

Comment:
Since JupyterLite is kernel-agnostic, `replite` could also be used for other languages such as Lua (https://github.com/jupyterlite/xeus-lua-kernel) or SQLite (https://github.com/jupyterlite/xeus-sqlite-kernel).

Which means the `micropip.install()` will not be available.

Maybe an alternative would be to provide a `silent` or `prerun` option and let the users pass the `micropip.install` code in the query string parameter?